hash/fmtowns_flop.xml

```diff
@@ -1635,6 +1703,26 @@ Zurukamashi Ver 2.0                                           Nikkonren Kikaku
 		</part>
 	</software>
 
+	<software name="shangrl2sp">
+		<description>Shangrlia 2 Special Disk</description>
```

Comment:
This title looks like a mistransliteration.

Reply:
Yeah, it's "wrong", but it *is* the official title. See https://vndb.org/v3183 for example.